api/tests/unit/infrastructure/exports/campaigns/campaign-profile-collection-result-line_test.js

```diff
@@ -0,0 +1,592 @@
+
+        const campaignParticipationResultData = {
+          id: 1,
+          isShared: true,
+          isCompleted: true,
+          createdAt: new Date('2019-02-25T10:00:00Z'),
+          sharedAt: new Date('2019-03-01T23:04:05Z'),
+          userId: 123,
+          participantFirstName: 'Juan',
+          participantLastName: 'Carlitos',
+          division: '5me',
+          studentNumber: 'studentNumber',
+        };
+
+        const csvExcpectedLine = `"${organization.name}";` +
+        `${campaign.id};` +
+        `"${campaign.name}";` +
+        `"${campaignParticipationResultData.participantLastName}";` +
+        `"${campaignParticipationResultData.participantFirstName}";` +
+        '"Oui";' +
+        '2019-03-01;' +
+        '13;' +
+        '"Non";' +
+        '0;' +
+        '1;' +
+        '9;' +
+        '0;' +
+        '4' +
+        '\n';
+
+        //when 
+        const line = new CampaignProfileCollectionResultLine(campaign, organization, campaignParticipationResultData, competences, placementProfile);
+
+        //then
+        expect(line.toCsvLine()).to.equal(csvExcpectedLine);
+      });
+    });
+
+    context('When organization is SCO and managing students', () => {
+      beforeEach(() => {
+        organization.isSco = true;
+        organization.isManagingStudents = true;
+      });
+
+      context('when the participant does not have a division', () => {
+        it('should return the csv with empty division', () => {
+          //given
+          sinon.stub(PlacementProfile.prototype, 'isCertifiable').returns(false);
+          sinon.stub(PlacementProfile.prototype, 'getCertifiableCompetencesCount').returns(0);
+
+          campaign.id = 123;
+          campaign.name = 'test';
+          organization.name = 'Umbrella';
+
+          const campaignParticipationResultData = {
+            id: 1,
+            isShared: true,
+            isCompleted: true,
+            createdAt: new Date('2019-02-25T10:00:00Z'),
+            sharedAt: new Date('2019-03-01T23:04:05Z'),
+            userId: 123,
+            participantFirstName: 'Juan',
+            participantLastName: 'Carlitos',
+            division: null,
+          };
+
+          const csvExcpectedLine = `"${organization.name}";` +
+        `${campaign.id};` +
+        `"${campaign.name}";` +
+        `"${campaignParticipationResultData.participantLastName}";` +
+        `"${campaignParticipationResultData.participantFirstName}";` +
+        '"";' +
+        '"Oui";' +
+        '2019-03-01;' +
+        '13;' +
+        '"Non";' +
+        '0;' +
+        '1;' +
+        '9;' +
+        '0;' +
+        '4' +
+        '\n';
+
+          //when 
+          const line = new CampaignProfileCollectionResultLine(campaign, organization, campaignParticipationResultData, competences, placementProfile);
+
+          //then
+          expect(line.toCsvLine()).to.equal(csvExcpectedLine);
+        });
+
+        context('when the participant has a division', () => {
```

Comment:
il faut fermer le contexte précédent, on n'est plus dans le cas "when the participant does not have a division"